packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1224,10 +1232,6 @@ export function useGlobalStylesOutputWithConfig( mergedConfig = {} ) {
 		return !! getSettings().disableLayoutStyles;
 	} );
 
-	const blockContext = useContext( BlockContext );
```

Comment:
This was the bug, who says that useGlobalStylesOutput need to be run within "BlockContext" and why?